lib/4_filter_numbers.rb

```diff
@@ -5,5 +5,6 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def filter_numbers(arr)
-  # Write your code here.
+    arr.delete_if {|z| z > 9 }    
```

Comment:
As it's written, what result would I get if I ran `filter_numbers([1, 2, 3, 9.5, 11, 15.5])`?